### DISCUSSION TITLE
Version 0.18.2 input parameters

### DISCUSSION THREAD

**Discussion by omermar** [mapequation/infomap#350] [General]
Hi.
I am trying to reconstruct some of my old analysis from this paper: [Predicting Protospecies (2018)](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0192871), where I have use Infomap to detect communities in lognormal networks.
The version I ran was 0.18.2, on UNIX, and the input parameters were "-d -k -z". 
I guess -d is for directed network, but what are the -k & -z? 
Much appriciated! Omer.

**Comment by danieledler** [comment]
Hi Omer!

The `-k` flag includes self-links, but that is included by default now (need to use `--no-self-links` to exclude them).

The `-z` flag sets zero-based node indexing, but that is no longer needed as the current Infomap no longer assumes consecutive ids (no gaps) starting from 1, but allows gaps.

If you have any other issues replicating your work, please comment again.
